### PR TITLE
RHCLOUD-27274 | feature(query): used event types per organization

### PR DESCRIPTION
**Pull request by MikelAlejoBR**
Lists the used event types per organization.

## Jira ticket
[[RHCLOUD-27274]](https://issues.redhat.com/browse/RHCLOUD-27274)

### CODE REVIEW

File: .rhcicd/floorist/floorplan.yaml [hunk 1]
```diff
@@ -152,6 +152,25 @@ objects:
...
+              ON etb.event_type_id = event_type.id
+          INNER JOIN
+            behavior_group_action AS bga
+              ON bga.behavior_group_id = etb.behavior_group_id
+          INNER JOIN
+            behavior_group
+              ON behavior_group.id = bga.behavior_group_id
+          GROUP BY
+            event_type.display_name,
+            behavior_group.org_id
```
Comment (gwenneg) [inline .rhcicd/floorist/floorplan.yaml:173]: The query does not contain any aggregate function so there's no need for a `GROUP BY` clause.
```suggestion
```

Comment (MikelAlejoBR) [reply .rhcicd/floorist/floorplan.yaml:173]: True, good catch!

**Review by gwenneg — APPROVED** [review]
LGTM

**Comment by gwenneg** [comment]
@MikelAlejoBR This PR will need a rebase.

**Comment by MikelAlejoBR** [comment]
Done!

File: .rhcicd/floorist/floorplan.yaml [hunk 2]
```diff
@@ -182,6 +182,26 @@ objects:
...
```
Comment (gwenneg) [inline .rhcicd/floorist/floorplan.yaml:204]: This needs to be removed again 😄 
```suggestion
```

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**